ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -60,15 +60,35 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: azure-ipi-workers-rhel8-f28
-  cron: 6 9 3 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f14
+  cron: 26 16 6,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FILTERS_ADDITIONAL: ~TELCO&
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: azure-ipi-workers-rhel8-f14
+  cron: 31 22 14,30 * *
   steps:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       FILTERS_ADDITIONAL: ~TELCO&
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.12 and @azure-ipi'
```

Comment:
cc @liangxia to help confirm if we could remove the tags here

Reply:
it seems yes, let me remove them.

Reply:
if you had confirm it, no concern from my side, could u pls also remove the tags in cpou jobs together, thanks

Reply:
sure

Reply:
@JianLi-RH you can ignore this one, i just drop it in another pr